source/icu.net.tests/SetUpFixture.cs

```diff
@@ -1,7 +1,5 @@
-// Copyright (c) 2017-2022 SIL International
+// Copyright (c) 2017-2025 SIL Global
 // This software is licensed under the MIT license (http://opensource.org/licenses/MIT)
-
-using System;
```

Comment:
Guessing this might be needed in the #if NETFRAMEWORK case.

Reply:
Thanks! I caught that one too, but there's another I removed that's necessary for ubuntu and still evading me...

Reply:
I changed the pr to draft, and I'll just roll back all the `using` changes if I can't get it sorted quickly.